### PR TITLE
Add EMG-based hand recognition dataset and example

### PR DESCRIPTION
**Pull request by pereverges**

### CODE REVIEW

File: examples/emg_based_hand_gesture.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            samples_hv = model.encode(samples)
+            model.classify.weight[labels] += samples_hv
+
+        model.classify.weight[:] = F.normalize(model.classify.weight)
+
+    accuracy = metrics.Accuracy()
+
+    with torch.no_grad():
+        for samples, labels in tqdm(test_ld, desc="Testing"):
+            samples = samples.to(device)
```
Comment (mikeheddes) [inline examples/emg_based_hand_gesture.py:103]: Duplicate line for labels `labels = labels.to(device)`

File: hdc/metrics.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import torch
```
[the remark this answers is not in the document]
Comment (mikeheddes) [reply hdc/metrics.py:1]: Can you still remove this file from the PR?
